src/pages/Environments.js

```diff
@@ -452,7 +451,11 @@ const Environments = () => {
                 content: div({ style: { padding: '0.5rem' } }, [
                   div([span({ style: { fontWeight: 600 } }, ['Name: ']), name]),
                   runtime && div([span({ style: { fontWeight: 600 } }, ['Runtime: ']), runtime.runtimeName]),
-                  app && div([span({ style: { fontWeight: 600 } }, ['Galaxy: ']), app.appName])
+                  app && div([
+                    span({ style: { fontWeight: 600 } },
+                      [app.appType ? `${_.capitalize(app.appType)}: ` : 'Galaxy: ']
+                    ), app.appName
+                  ])
```

Comment:
Just noticed this other place where Galaxy was hardcoded. Once the Leo changes merge, this can be simplified to just be `${_.capitalize(app.appType)}: `

![image](https://user-images.githubusercontent.com/484484/139946143-986e638f-9cfe-45d9-85aa-13d03403bb52.png)